src/server/pps/pretty/pretty.go

```diff
@@ -186,6 +189,78 @@ Job Counts:
 	return nil
 }
 
+// PrintDatumInfoHeader prints a file info header.
+func PrintDatumInfoHeader(w io.Writer) {
+	fmt.Fprint(w, "ID\tSTATUS\tTIME\t\n")
+}
```

Comment:
Why not split the time up a bit more here between D/L, U/L and process time? With only 3 fields there should be plenty of space.